playbooks/roles/xapi_bridge/templates/xapi_bridge.conf

```diff
@@ -0,0 +1,13 @@
+[program:xapi_bridge]
```

Comment:
This .conf is for a Supervisor program still, right?  Shouldn't we get rid of this if we are going to use systemd?

Reply:
@bryanlandia yeah we're not using supervisor for managing the process. But I still wanted to keep this here to try a few things out. I'll remove this and keep it in a different branch.